src/app/shared/models/appServer.ts

```diff
@@ -3,7 +3,7 @@ export interface FcmNotificationDto {
   body: string
```

Comment:
can you update the file name to same naming convention as other files `app-server`?

Reply:
> LGTM. Thanks for the changes. A couple of structural suggestions.
> 
> PS: There is also a property called `sourceType` in Notification entity on appserver. You can also filter by that, so we only cancel notifications scheduled from the aRMT app. But we can add this later (right now only the app is scheduling so its fine to remove all).

Okay I see I'll add it later. Thanks.